index.php

```diff
@@ -20,8 +43,41 @@
 </head>
 
 <body>
+    <!-- nav-bar -->
+    <div class='navBar'>
+        <div class='leftNav'>
+            <a class='navLink'>MyRecords</a>
+        </div>
+        <div class='rightNav'>
+            <a class='navLink'>+ Record</a>
+            <a class='navLink'>Archive</a>
+        </div>
+    </div>
+    <!-- nav-bar -->
+    <!-- record display -->
+    <div class='flexConatiner'>
+        <?php
+        foreach ($allRecords as $record) {
 
-<h1>Website Template</h1>
+            $genreInfo = $recordModel->getGenreByID($record->genre_id);
+            $genreName = $genreInfo['name'];
 
+            echo
+            "<div class='albumContainer'>
+            <img src='$record->img' alt='$record->album_name' width='300' height='300' >
+            <div class='albumStats'>
+            <p class='smallCopy'><strong>Album:</strong> $record->album_name</p>
+            <p class='smallCopy'><strong>Artist:</strong> $record->artist_name</p>
+            <p class='smallCopy'><strong>Year of realse:</strong> $record->release_year</p>
+            <p class='smallCopy'><strong>Genre:</strong> $genreName</p>
+            <p class='smallCopy'><strong>Score:</strong> $record->score/10</p>
+            </div>
+            </div>";
+        }
```

Comment:
You should make this into a function that displays all records, you give it the array of records and it **returns** the string of HTML needed to display them all.

It could be useful on your archive page, and from a practice perspective means that you need to write some unit tests for it! (Since this would be your only function that doesn't directly interact with a database)